plasmapy/classes/sources/openpmd_hdf5.py

```diff
@@ -166,3 +165,9 @@ def is_datasource_for(cls, **kwargs):
                 openPMD = False
 
         return openPMD
+
+
+class DataStandardError(Exception):
+    """An exception for when the structure of a data file does not match the
+    expected standard."""
+    pass
```

Comment:
This makes me hopeful for the future...for a time when plasma physicists have standard formats for our data!